### PR TITLE
Compile OL3 and OL3-Cesium together

### PR DESCRIPTION
**Pull request by gberaudo**
OL3-Cesium used to be compiled against OL3 externs.
This method had many shortcomings and is no more supported upstream.

This PR updates geoadmin accordingly.
It also allows again to compile the minified and unminified versions of Cesium independantely.

Test links [built](https://mf-geoadmin3.dev.bgdi.ch/gberaudo_bundled_ol3cesium/?dev3d=true&lang=en&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bafu.wrz-wildruhezonen_portal,ch.swisstopo.swisstlm3d-wanderwege&layers_visibility=false,false,false,false&layers_timestamp=18641231,,,&lon=8.22546&lat=42.69089&elevation=517826&heading=360.000&pitch=-44.169) [src](https://mf-geoadmin3.dev.bgdi.ch/gberaudo_bundled_ol3cesium/src/?dev3d=true&lang=en&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bafu.wrz-wildruhezonen_portal,ch.swisstopo.swisstlm3d-wanderwege&layers_visibility=false,false,false,false&layers_timestamp=18641231,,,&lon=8.22546&lat=42.69089&elevation=517826&heading=360.000&pitch=-44.169)

### CODE REVIEW

**Comment by oterral** [comment]
The unminified version of Cesium is used in prod mode.

**Comment by oterral** [comment]
In `src` mode directive files are loaded tree times each.

**Comment by gberaudo** [comment]
Thanks @oterral, it should be fixed now.
Removing and redeploying the branch fixed the bug with the directives.

I had to rename the angular module and another symbol, please have a look to [this commit message](https://github.com/geoadmin/mf-geoadmin3/commit/a7d215b6b8d50c10ee520737796acea5c14d7942).

File: Makefile [hunk 1]
```diff
@@ -270,13 +268,13 @@ prd/lib/: src/lib/d3-3.3.1.min.js \
 prd/lib/build.js: src/lib/jquery-2.0.3.min.js \
 		    src/lib/bootstrap-3.3.1.min.js \
 		    src/lib/moment-with-customlocales.min.js \
-		    src/lib/typeahead-0.9.3.min.js src/lib/angular.min.js \
+		    src/lib/typeahead-0.9.3.min.js src/lib/angular.js \
```
Comment (oterral) [inline Makefile]: Why did you use the uncompressed angular ?  
Angular doesn't  seems to be compressed in the final build.js file.

Comment (gberaudo) [inline Makefile]: @oterral, it was a leftover from a debugging session.
Thanks for finding it. It is is now fixed and I also updated ol3-cesium to latest commit.

**Comment by oterral** [comment]
@gberaudo on dev mode (with cache disabled in dev tools)  it seems the files are still loaded 2 or 3 times each.

**Comment by gberaudo** [comment]
@oterral, that is very strange. Have you tested with the provided links or a different urls?
On my machine I do not see duplication.

**Comment by oterral** [comment]
With your link and on my environment

**Comment by oterral** [comment]
Only on Chrome

**Comment by gberaudo** [comment]
Even stranger: I am testing on chromium and I had one tab with duplication and the other without.
I reset chromium and open two tabs, I see no duplication.

**Comment by gberaudo** [comment]
Argh, duplication again.
It seems that `ctrl+R`-> no duplication, but simply pressing `enter` on the URL bar cause duplication.

**Comment by gberaudo** [comment]
@oterral, `coup de théatre`, the issue already occurs in master and thus is unrelated to this PR.
See issue #2951. I think we can now merge this PR, thank you for the review.

**Comment by oterral** [comment]
Damned!!!! thanks @gberaudo for finding this.
ltgm we can merge I think.

**Comment by gjn** [comment]
Very nice work @gberaudo . It's a much needed consolidation. Thanks alot.

**Pull request merged**